src/main/kotlin/interactor/GetCityThatHasLowestCostOfFoodInteractor.kt

```diff
@@ -0,0 +1,55 @@
+package interactor
+
+import jdk.internal.net.http.common.Pair
+import model.CityEntity
+
+
+/**
+write a function that return the city name that has the lowest cost of
+food (rice , loaf of bread , chicken , eggs,cheese , beef)
+let your function return list of paris that has city name and the cost
+data quality should be high
+ * **/
+ class GetCityThatHasLowestCostOfFoodInteractor(
+    private val dataSource: CostOfLivingDataSource,
+) : CostOfLivingDataSource {
```

Comment:
Remove the implementation of the CostOfLivingDataSource interface
```suggestion
)  {
```